.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!---
+    Thank you for contributing to Unity.
+    To help us process this pull request we recommend that you add the following information:
+     - Summary and list of changes in the pull request,
+     - Issue(s) related to the changes made such as GitHub or Jira,
+     - Manual testing scenarios if available
+    Fields marked with (*) are required. Please don't remove the template.
+-->
+### Description (*)
+<!---
+    Please provide a description of the changes proposed in the pull request.
+    Letting us know what has changed and why it needed changing will help us validate this pull request.
+-->
+### Related Pull Requests
+<!-- related pull request placeholder -->
+### Issue Number(s) (*)
+<!---
+    Provide a list of fixed issues from Jira (GOMPS-ticketnumber) or GitHub (#issuenumber).
+    This helps us understand the reasoning behind this change, what it fixes, feature being added, etc.
+-->
+Fixes issue(s):
+### Manual testing scenarios
+<!---
+    Please provide a set of unambiguous steps to test the proposed code change.
+    Giving us manual testing scenarios will help with the processing and validation process.
+-->
+1. ...
+2. ...
+### Questions or comments
+<!---
+	If relevant, here you can ask questions or provide comments on your pull request for the reviewer
+	For example if you need assistance with writing tests or would like some feedback on one of your development ideas or for the reviewer to focus on a particular area of the code.
+-->
+### Contribution checklist
```

Comment:
Curious about this. Does the author fill these in or do they serve more of mental notes before creating PR? Asking because it shows up on the preview.

Reply:
Can be used as a checklist prior to merging, could edit these options or just remove if not helpful.